code/obj/flock/furniture.dm

```diff
@@ -236,17 +238,21 @@
 /obj/lattice/flock/New()
 	..()
 	setMaterial("gnesis")
-	src.AddComponent(/datum/component/flock_protection, FALSE, FALSE, TRUE)
+	src.AddComponent(/datum/component/flock_protection, report_attack=FALSE)
 
 /obj/lattice/flock/attackby(obj/item/C as obj, mob/user as mob)
 	if (istype(C, /obj/item/tile))
 		var/obj/item/tile/T = C
 		if (T.amount >= 1)
+			if(SEND_SIGNAL(src, COMSIG_FLOCK_ATTACK, user))
+				return
```

Comment:
Not sure if reporting for placing tiles is needed, don't think it's really an attack, feels more closer a repurposing type of thing similar to humans deconstructing flock furniture